packages/react/runtime/src/lifecycle/patch/snapshotPatchApply.ts

```diff
@@ -94,7 +100,7 @@ export function snapshotPatchApply(snapshotPatch: SnapshotPatch): void {
           const cssId: number = snapshotPatch[++i] ?? 0;
           const entryName: string | undefined = snapshotPatch[++i];
 
-          if (!snapshotManager.values.has(uniqID)) {
+          if (!snapshotManager.values.has(entryUniqID(uniqID, entryName))) {
```

Comment:
We should also update the check at `createSnapshot`, which could reduce the patch size.

https://github.com/lynx-family/lynx-stack/blob/1b7c028e8ad3de6074170ebb5c7f731396463421/packages/react/runtime/src/snapshot.ts#L197